src/Components/Dialogs/dialogsContainer.jsx

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import { addMessageActionCreator, updateMessageActionCreator } from '../../redux/redux-state';
+
+import Dialogs from './dialogs';
+
+const mapStateToProps = (state) => ({
+  dialogsPages: state.dialogsPages,
+});
+const mapDispatchToProps = (dispatch) => ({
+  AddNewMessage: () => {
```

Comment:
Consider renaming such methods to have a lowercase letter at the start of the name.

Reply:
sure